ita-social-projects/GreenCityUBS
# [UBS] - change response in ManagementEmployeeController for save and update methods #5257

# GreenCityUBS PR
https://github.com/ita-social-projects/GreenCity/issues/5257

## Summary Of Changes :fire:
Changed response in ManagementEmployeeController

## Issue Link :clipboard:
_&lt;Link to the issue&gt;_

## Added
Added new response dto (EmployeeWithTariffsDto) in ManagementEmployeeController for save and update methods
Added test for a new mapper

## Changed
* _&lt;detail item of what was changed&gt;_
* _&lt;describe functionality that was changed&gt;_

## Deleted
* _&lt;detail item of what was removed&gt;_

## How to test :clipboard:
It can be tests via swagger in management-employee-controller, methods save and update.
The response should contain EmployeeDtoWithTariffs

# PR Checklist Forms

_(to be filled out by PR submitter)_
